From ca87aacbfa894880c50c02a2019eed5fdb369251 Mon Sep 17 00:00:00 2001
In-Reply-To: <0181114090144.31412-1-avarab@gmail.com>
References: <0181114090144.31412-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 14 Nov 2018 09:14:10 +0000
Subject: [PATCH v2 0/2] rebase.useBuiltin doc & test mode
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

I was missing a "=" in the t/README docs in 2/2 in v1. Also move the
docs around slightly so this & my gettext series don't have conflicts.

*** BLURB HERE ***

Ævar Arnfjörð Bjarmason (2):
  rebase doc: document rebase.useBuiltin
  tests: add a special setup where rebase.useBuiltin is off

 Documentation/config/rebase.txt | 14 ++++++++++++++
 builtin/rebase.c                |  5 ++++-
 t/README                        |  4 ++++
 3 files changed, 22 insertions(+), 1 deletion(-)

Range-diff:
1:  a5508195c6 ! 1:  ca87aacbfa tests: add a special setup where rebase.useBuiltin is off
    @@ -29,12 +29,13 @@
      --- a/t/README
      +++ b/t/README
     @@
    - index to be written after every 'git repack' command, and overrides the
    - 'core.multiPackIndex' setting to true.
    + GIT_TEST_PRELOAD_INDEX=<boolean> exercises the preload-index code path
    + by overriding the minimum number of cache entries required per thread.
      
    -+GIT_TEST_REBASE_USE_BUILTIN<boolean>, when false, disables the builtin
    -+version of git-rebase. See 'rebase.useBuiltin' in git-config(1).
    ++GIT_TEST_REBASE_USE_BUILTIN=<boolean>, when false, disables the
    ++builtin version of git-rebase. See 'rebase.useBuiltin' in
    ++git-config(1).
     +
    - Naming Tests
    - ------------
    - 
    + GIT_TEST_INDEX_THREADS=<n> enables exercising the multi-threaded loading
    + of the index for the whole test suite by bypassing the default number of
    + cache entries and thread minimums. Setting this to 1 will make the
-- 
2.19.1.1182.g4ecb1133ce

